packages/web-wallet/index.html

```diff
@@ -12,6 +12,5 @@
   </head>
   <body>
     <div id="root"></div>
-    <script type="module" src="/src/main.tsx"></script>
```

Comment:
We don't need this?

Reply:
No, now we're using webpack and it injects it